packages/amplify-provider-awscloudformation/lib/amplify-service-migrate.js

```diff
@@ -0,0 +1,110 @@
+const fs = require('fs-extra');
+const configurationManager = require('./configuration-manager');
+const { getConfiguredAmplifyClient } = require('../src/aws-utils/aws-amplify');
+const { checkAmplifyServiceIAMPermission } = require('./amplify-service-permission-check');
+const { storeCurrentCloudBackend } = require('./push-resources');
+const constants = require('./constants');
+
+async function run(context) {
+  const projectDetails = context.amplify.getProjectDetails();
+  const awsConfig = await configurationManager.getAwsConfig(context);
+
+  const amplifyClient = await getConfiguredAmplifyClient(context, awsConfig);
+  if (!amplifyClient) {
+    // This happens when the Amplify service is not available in the region
+    const message = `Amplify service is not available in the region ${awsConfig.region ? awsConfig.region : ''}`;
+    context.print.error(message);
+    throw new Error(message);
```

Comment:
We shouldn't throw an error for this case right? We should go ahead and provision the AWS resources without creating the Amplify app? cc @swaminator

Reply:
We should, because this workflow is explicitly invoked by the customer. And we should let the customer know it's not possible to execute this workflow in the particular region, because Amplify Console service is not available in the configured region. Throwing error also stops the actions following this function, which is our intention in those regions.